plugins/doc_fragments/ipamodule_base_docs.py

```diff
@@ -20,6 +20,10 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
```

Comment:
This is a note for another PR: it seems this should be included in the `nem_module` templates, shouldn't it?

Reply:
Done